File: C Programming/Assignments/Assignment 6/Palindromeno.c
//Write a program to check 3 digit no. is palindrome or not using Function
int palindromeno();

int main()
{
	int res=palindromeno();
	if(res==0)
	{
		printf("Palindrome");
	}
	else
	{
		printf("Not palindrome");
	}	
		
}

int palindromeno()
{
	int no=121;
	int lno=no%10;
	int fno=no/100;
	if(lno==fno)
	return 0;
	else
	return 1;

}

File: JAVA/Assignment 1/Palindromeno.java
import java.util.Scanner;
public class Palindromeno
{
public static void main(String args[])
{
Scanner sc=new Scanner(System.in);
	int no,lno,fno;
	System.out.printf("Enter no=");
	no=sc.nextInt();
	lno=no%10;
	fno=no/100;
	if(lno==fno)
	{
		System.out.printf("%d is Palindrome no",no);
	}
	else
	{
	System.out.printf("%d is Not Palindrome no",no);
	}
}
}